ckanext/versioning/logic/helpers.py

```diff
@@ -118,3 +116,16 @@ def get_dataset_current_revision(dataset_name):
     backend = get_metastore_backend()
 
     return backend.fetch(dataset_name).revision
+
+def get_dataset_version(package_id, revision_ref):
```

Comment:
BTW: can we refactor all the `version` terminology to something like `revision_tag` before we have final so reduce future confusion.